sys/include/util_macros.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys
+ * @{
+ *
+ * @file
+ * @brief       Miscellaneous helper macros
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ */
+
+#ifndef UTIL_MACROS_H
+#define UTIL_MACROS_H
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/* KB, MB, GB */
+#define KB(x)   ((x) << 10)
```

Comment:
Does this need some casting to not default to `int`? what does avr-gcc make of `MB(10)`?